### PR TITLE
Release 11/22/2022

### PR DESCRIPTION
**Pull request by johnhalbert**
Re #102

### CODE REVIEW

File: command/command.go [hunk 1]
```diff
@@ -297,9 +297,6 @@ func handleCommitRequest(cache *cache.Cache, commitMsg *sync_pb.CommitMessage, c
 		entryRsp.ResponseType = &rspType
 		entryRsp.IdString = aws.String(entityToCommit.ID)
 		entryRsp.Version = entityToCommit.Version
-		entryRsp.ParentIdString = entityToCommit.ParentID
```
Comment (AlexeyBarabash) [inline command/command.go:300]: why we had to remove it?

File: schema/protobuf/sync_pb/app_specifics.proto [hunk 2]
```diff
@@ -80,24 +80,22 @@ message AppSpecifics {
   // This describes how the extension should be launched.
   optional LaunchType launch_type = 5;
 
-  // This is the url of a bookmark app. If this exists, the app is a bookmark
-  // app.
-  optional string bookmark_app_url = 6;
+  // DEPRECATED. See https://crbug.com/1233303.
```
Comment (AlexeyBarabash) [inline schema/protobuf/sync_pb/app_specifics.proto:83]: updated proto files :+1:

**Comment by AlexeyBarabash** [comment]
I have verified the branch `master` (now at `https://sync-v2.brave.software/v2`) works good with the client from https://github.com/brave/brave-core/pull/13562

**Comment by AlexeyBarabash** [comment]
changes look good to me at that part which I was able to understand go and server stuff
so I don't put approve mark and defer it @yrliou

File: command/command.go (hunk 1 above)
Comment (johnhalbert) [reply command/command.go:300]: I updated protobuf schemas from chromium.  This field was updated to be marked reserved and was dropped from the generated Go code.

File: .gitignore [hunk 3]
```diff
@@ -13,3 +13,11 @@
...
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
+# vim files
+*.swp
+*.swo
+
+# Temp files
+
+tmp/
```
Comment (fmarier) [inline .gitignore:23]: nit: if you're looking to exclude only the `tmp` directory at the top level of the git repo, then you can use `/tmp/` instead. Using `tmp/` will match a `tmp` directory anywhere in the repo (e.g. `foo/bar/tmp`).

File: datastore/sync_entity.go [hunk 4]
```diff
@@ -263,6 +293,164 @@ func (dynamo *Dynamo) InsertSyncEntitiesWithServerTags(entities []*SyncEntity) e
 	return nil
 }
 
+// DisableSyncChain marks a chain as disabled so no further updates or commits can happen
```
Comment (fmarier) [inline datastore/sync_entity.go:296]: Do we keep track here of the reason why it was disabled (i.e. user requested deletion)?

The spec [lists three possible reasons](https://docs.google.com/document/d/1X7glDHmeIjgipcPWn__S9rSxrI39P9rNw0RmuEuC3BI/edit#heading=h.7vqo8iomb0b3) for a disabled account: expired, deleted, passphrase reset

Only one of these is being implemented right now, but we should probably start keeping track of this data now.

**Comment by yrliou** [comment]
FYI my comments are left in #102 directly.

**Comment by yrliou** [comment]
We need to make sure CI is passed before merging.

**Comment by mihaiplesa** [comment]
Please have a look at the lint/tests issues reported by GitHub Actions.

**Review by yrliou — APPROVED** [review]
++ from my side, but please make sure to get the approval from security team before merging, thanks!

**Comment by johnhalbert** [comment]
> ++ from my side, but please make sure to get the approval from security team before merging, thanks!

Spoke with @diracdeltas who was able to verify with @fmarier that security has signed off on this.

**Comment by yrliou** [comment]
@johnhalbert which chromium version did you use in this PR for those updated protocols? Please help update the README with the chromium version this repo is currently using.

**Pull request merged**